pyoverkiz/clients/somfy_local.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import annotations
+
+from attr import define, field
+
+from pyoverkiz.clients.overkiz import OverkizClient
+
+
+@define(kw_only=True)
+class SomfyLocalClient(OverkizClient):
+
+    _ssl: bool = field(default=False, init=False)
+    token: str = field(repr=lambda _: "***")
+
+    async def _login(self) -> bool:
```

Comment:
How can we verify if the local login works?

Reply:
We can call any command to see if it works. 
But I don’t think we should put a check here. There is no login endpoint for the local api, and so we should not call it.